### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by v411e**
Hey, I added some docs for docker-compose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ For more information, questions, or just the use of the hosted version, you can
...
 ## Usage
+The tracked repositories can be configured in multiple ways.
+
+### Bot-command
+Open a conversation with the bot and type `!github <username>`.
+This would track all repositories starred by the defined user.
+
+### State event
+A more fine-grained approach is to save the tracked entities in the room state of a room you share with the bot.
+Send a state event (e.g. via Element with `\devtools`) containing all the things you want to track. These can be easily updated with another state event.
```
Comment (ananace) [inline README.md]: Should be `/devtools`

File: README.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ For more information, questions, or just the use of the hosted version, you can
 ![Example image](https://i.imgur.com/iAP1rMs.png)
 
 ## Usage
+The tracked repositories can be configured in multiple ways.
+
+### Bot-command
+Open a conversation with the bot and type `!github <username>`.
```
Comment (ananace) [inline README.md]: I'd word this as something more like "Send the message `!github <username>` to the bot" instead, "open a conversation and type" sounds far too technical for what it is

**Review by ananace — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (v411e) [reply README.md]: Whoops that's right, sorry :)

**Comment by ananace** [comment]
Thanks

**Pull request merged**